scapy/route.py

```diff
@@ -36,7 +36,7 @@ def __repr__(self):
             rt += "%-15s %-15s %-15s %-15s %-15s\n" % (ltoa(net),
                                               ltoa(msk),
                                               gw,
-                                              iface,
+                                              (iface if not WINDOWS else iface.name),
```

Comment:
If you don't mind, could you reverse the expression so that it reads `iface.name if WINDOWS else iface`?